### ISSUE TITLE
Set up CI to publish image to dockerhub

### ISSUE THREAD

**Issue by silentninja** [mathesar-foundation/mathesar#2040]
## Problem
Our current docker image fetches our repo and builds it during the build process which takes a lot of time to get Mathesar up and running. Moreover, the user also needs to clone our repo and start the build process. 

## Proposed solution
It would a good  improvement if we could have a pre-built docker image 

## Additional context

- Blocked by #2032 
- Related to #2034

**Comment by dmos62** [comment]
In sync with Brent, we realised that this is a nice-to-have and we're deprioritising it (bumping it out of must-haves of Cycle 6).

**Comment by kgodey** [comment]
Closing this, too old, requirements out of date with current installation setup. We can open a new issue if we need this functionality in the future.
